packages/health-check/src/checks/critical.js

```diff
@@ -195,6 +195,7 @@ async function blockerHealthCheck(done) {
   done({ name: "blocker", time: calculateElapsedTime(time), ...data });
 }
 
+// genericAccessCheck
```

Comment:
Let's remove this from the PR once this is ready to be merged

```suggestion
```

Reply:
These were added to prove that the linters are properly triggered.